src/index.ts

```diff
@@ -467,6 +467,7 @@ export async function startAnalysis<
 
                     // eslint-disable-next-line require-atomic-updates
                     emulatorProcess = execa('emulator', cmdArgs);
+                    await platform.waitForDevice();
```

Comment:
It turns out that on my Macbook at least, 20 tries are not enough to wait for the emulator to start, I need about 100.

Reply:
Static timeouts across different computers will always be a problem… But that should be fixed in appstraction, shouldn't it?

Reply:
I thought so first, too. But actually, I would want it to wait for a device much shorter than for a lenghty emulator startup.

Reply:
And since we know we wait for an emulator here, we should extend the timeout, shouldn‘t we?

Reply:
I still feel like there should be a more idiomatic way of waiting for an emulator in appstraction, but sure.